apps/server/src/shared/testing/factory/domainobject/board/column-board.do.factory.ts

```diff
@@ -26,3 +27,18 @@ export const columnBoardFactory = ColumnBoardFactory.define(ColumnBoard, ({ sequ
 		isVisible: true,
 	};
 });
+
+export const createColumnBoardWithColumn = ColumnBoardFactory.define(ColumnBoard, ({ sequence }) => {
+	return {
+		id: new ObjectId().toHexString(),
+		title: `column board #${sequence}`,
+		children: [columnFactory.build()],
+		createdAt: new Date(),
+		updatedAt: new Date(),
+		context: {
+			type: BoardExternalReferenceType.Course,
+			id: new ObjectId().toHexString(),
+		},
+		isVisible: true,
+	};
+});
```

Comment:
Instead of creating a new factory, we can modify the existing one like this (see code below) and use it like this `columnBoradFactory.build({ children: [columns...] })`:

```suggestion

export const columnBoardFactory = ColumnBoardFactory.define(ColumnBoard, ({ sequence, params }) => {
	return {
		id: new ObjectId().toHexString(),
		title: `column board #${sequence}`,
		children: params?.children ?? [],
		createdAt: new Date(),
		updatedAt: new Date(),
		context: {
			type: BoardExternalReferenceType.Course,
			id: new ObjectId().toHexString(),
		},
		isVisible: true,
	};
});
```